demisto_sdk/commands/validate/validators/IF_validators/IF109_invalid_required_field.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+from typing import Iterable, List, Union, cast
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.content_graph.objects.incident_type import IncidentType
+from demisto_sdk.commands.content_graph.objects.indicator_field import IndicatorField
+from demisto_sdk.commands.content_graph.objects.indicator_type import IndicatorType
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Union[IncidentField, IndicatorField, IncidentType, IndicatorType]
+
+
+class IsValidRequiredFieldValidator(BaseValidator[ContentTypes]):
+    error_code = "IF109"
+    description = "Checks if the incident field required field is valid."
+    rationale = "In case an incident field is required, newly added associated incident types should be new."
+    # error_message = "{0}"
```

Comment:
error message?

Reply:
No need to have it, since we have different errors, which are returned in the result